### PR TITLE
Annotations on plot

### PR DESCRIPTION
**Pull request by birdage**
Annotations on plot

### CODE REVIEW

**Comment by birdage** [comment]
screen shot 2015-08-17 at 5 03 52 pm

**Comment by DanielJMaher** [comment]
@FBRTMaka YOu wanna review this at all?

**Comment by FBRTMaka** [comment]
@DanielJMaher sure can!

File: ooiui/static/css/common/toc_menu.css [hunk 1]
```diff
@@ -2,8 +2,14 @@
     max-height:80%;
 }
 #assetBrowser {
-    margin-top:25px;
```
Comment (FBRTMaka) [inline ooiui/static/css/common/toc_menu.css]: @birdage please checked this doesn't do anything strange on the data catalog page.

File: ooiui/static/css/common/toc_menu.css [hunk 2]
```diff
@@ -2,8 +2,14 @@
...
+    margin-top:5px;
 }
+
+.toc-header{
```
Comment (FBRTMaka) [inline ooiui/static/css/common/toc_menu.css]: @birdage agan, always remember that not ALL pages use the TOC identically. *winky face*

File: ooiui/templates/science/plotting.html [hunk 3]
```diff
@@ -434,17 +428,44 @@
...
+    //render the time series data
+    this.listenTo(this.collections.dataseries, 'reset', function(series) {
+      this.views.timeseriesView.collection = series;
+      this.views.timeseriesView.render();
+
+      self.collections.annotations.each(function(annotation,i) {  
+        self.views.timeseriesView.views.highchartsView.chart.xAxis[0].addPlotBand({
+                from: moment.utc(annotation.get('beginDT')),
+                to: moment.utc(annotation.get('endDT')),
+                color: '#FCFFC5',
```
Comment (FBRTMaka) [inline ooiui/templates/science/plotting.html]: @Bobfrat , @birdage lol may want to double check the removal of time data from the streams doesn't break this.

**Comment by FBRTMaka** [comment]
@DanielJMaher , @birdage ; just quick verifications of those items and should be good to go!

File: ooiui/static/css/common/toc_menu.css (hunk 1 above)
Comment (birdage) [inline ooiui/static/css/common/toc_menu.css]: its fine

File: ooiui/static/css/common/toc_menu.css (hunk 2 above)
Comment (birdage) [inline ooiui/static/css/common/toc_menu.css]: not sure what you mean by this lol

File: ooiui/templates/science/plotting.html (hunk 3 above)
Comment (birdage) [inline ooiui/templates/science/plotting.html]: not sure what you mean by this lol

**Comment by birdage** [comment]
merged with dans ok

File: ooiui/templates/science/plotting.html (hunk 3 above)
Comment (FBRTMaka) [inline ooiui/templates/science/plotting.html]: Oh sorry, it was just related to a PR that bob made.  If it all works after he did his change then it's good!

**Pull request merged**